src/env.cpp

```diff
@@ -307,37 +337,16 @@ NAN_METHOD(EnvWrap::sync) {
         return Nan::ThrowError("The environment is already closed.");
     }
 
-    Local<Function> callback = info[0].As<Function>();
-
-    EnvSyncData *d = new EnvSyncData;
```

Comment:
This was the only place where `EnvSyncData` was ever used, can you please delete it now that it's not used anymore?